projects/packages/forms/src/contact-form/class-contact-form-field.php

```diff
@@ -902,17 +902,64 @@ public function render_date_field( $id, $label, $value, $class, $required, $requ
 		}
 
 		Assets::register_script(
-			'grunion-frontend',
-			'../../dist/contact-form/js/grunion-frontend.js',
+			'jp-date-picker',
+			'../../dist/contact-form/js/date-picker.js',
```

Comment:
Looks like grunion-frontend.js only had a small bit of JS related to the old date picker. If we're not going use that any more, I assume we'll want to delete it?